dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/utils/TaskExecutionCheckerUtils.java

```diff
@@ -131,4 +138,23 @@ public static void downloadResourcesIfNeeded(StorageOperate storageOperate,
             }
         }
     }
+
+    private static void createDirectoryWithOwner(Path filePath, String tenant) {
+        if (Files.exists(filePath)) {
+            return;
+        }
+        try {
+            Files.createDirectories(filePath);
+            if (!OSUtils.isSudoEnable()) {
+                // we need to open sudo, then we can change the owner.
+                return;
```

Comment:
Is is better to throw a exception here?

Reply:
If we throw an exception here, it means we only support deploy with `sudo`, then this switch is meaningless. We don't need to throw exception here.